src/main/java/org/sonar/plugins/checkstyle/metadata/CheckstyleMetadata.java

```diff
@@ -242,7 +307,11 @@ public static boolean isSetter(Method method) {
      * @param name the name fetched from ModuleDetails
      * @return modifiedName
      */
-    private static String convertName(String name) {
+    private static String convertName(String moduleName) {
+        String name = moduleName;
+        if (!name.contains(CHECK_STRING)) {
+            name += CHECK_STRING;
```

Comment:
not all modules are `Checks`
Looks like this method is not only for Checks, so please update name of method to mention this.

Reply:
In sonar all modules are checks, unlike `eclipse-cs`

Reply:
module is term of checkstyle.
Sonar keep all filters in separate config area so user define them as xml chunks, so it kind of split modules in two groups, 
that is why you see only Checks.

but terminology wise we should not be misleading.
`getFullCheckName`

Reply:
Done